signup.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Signup</title>
+    <link rel="stylesheet" href="./css/signup.css">
+</head>
+<body>
+    <div class="content1">
+
+    </div>
+
+    <div class="content2">
+        <div class="frame">
+            <div class="frame1">
+                <img class="logo" src="./images/logo.svg" onclick="location.href='/'">
+                <div>
+                    <span class="comment1">이미 회원이신가요?</span> <a class="comment2" href="/signin.html">로그인 하기</a>
+                </div>
+        
+                <div class="emailframe">
+                    <span class="comment3">이메일</span>
+                    <input type="email" class="email" value="codeit@codeit.com">
+                </div>
+        
+                <div class="emailframe">
+                    <span class="comment3">비밀번호</span>
+                    <input class="email" value="codeit@codeit.com">
```

Comment:
위의 이메일 입력란처럼 type을 주면 좋을 것 같습니다!

```suggestion
                    <input type="password" class="email" value="codeit@codeit.com">
```

Reply:
넵!!